starfish/plot/util.py

```diff
@@ -0,0 +1,108 @@
+from typing import Any, Dict, Optional
+
+import matplotlib.axes
+import matplotlib.figure
+import matplotlib.pyplot as plt
+
+
+# TODO ambrosejcarr: need to scale font size with figure size
+# TODO ambrosejcarr: need a function to equalize plot axis sizes
+
+def set_default_arguments(dictionary: Optional[Dict]=None, **kwargs) -> Dict[str, Any]:
+    """
+    Non-overwriting kwargs setter. Sets arguments in an optional dictionary only if they are not
+    already present
+
+    Parameters
+    ----------
+    dictionary : Dict
+        dictionary to set keyword arguments in. Created if not provided
+    kwargs : Dict
+        keywords to add to the dictionary
+
+    Returns
+    -------
+    Dict :
+        Dictionary containing keywords
+
+    """
+    dictionary = {} if dictionary is None else dictionary
+    for key, value in kwargs.items():
+        if key not in dictionary:
+            dictionary[key] = value
+    return dictionary
+
+
+def dpi_correction(figure: Optional[matplotlib.figure.Figure]=None) -> float:
+    """
+    matplotlib.scatter size correction for DPI to ensure that points have constant size across
+    visible scales
+
+    Parameters
+    ----------
+    figure : Optional[matplotlib.figure.Figure]
+        Figure in which to scale scatterplot size. If not provided, defaults to current figure.
+
+    Returns
+    -------
+    float :
+        multiplier for matplotlib.plot.scatter markersize kwarg to ensure correct size in pixels
+
+    """
+    if figure is None:
+        figure = plt.gcf()
+    point_multiplier = 72. / figure.dpi
```

Comment:
where does the 72 come from?

Reply:
This is apparently the default scaling parameter between pixels and DPI in matplotlib. I will add a comment. Thanks for the catch.